packages/builders/package.json

```diff
@@ -24,6 +24,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@agoric/ertp": "^0.16.2",
+    "@agoric/fast-usdc": "0.1.0",
```

Comment:
help? this reportedly introduces a cycle:

```
lerna ERR! ECYCLE agoric -> @agoric/cosmic-swingset -> @agoric/builders -> @agoric/fast-usdc -> agoric
```
-- [ci log](https://github.com/Agoric/agoric-sdk/actions/runs/11845084122/job/33009893297?pr=10477#step:4:21)


I don't get it. where does the `@agoric/fast-usdc -> agoric` dep come from? I don't see it in packages/fast-usdc/package.json

Reply:
Looks like a Lerna bug with its "ignore" list used in `lint-with-types.sh`. I expect 3cad269d681b2b6149f3dde988801225fa8d3e72 will fix it